api/src/main/java/uk/gov/dhsc/htbhf/claimant/SchedulerConfig.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.dhsc.htbhf.claimant;
+
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+import javax.sql.DataSource;
+
+@Configuration
+@EnableScheduling
+// default max lock time is required and non configurable
+@EnableSchedulerLock(defaultLockAtMostFor = "PT60M")
```

Comment:
Can we just say what PT60M stands for in the JavaDoc?

Reply:
Done.